AstarteDeviceSDKCSharp/Crypto/AstarteMutualTLSParametersFactory.cs

```diff
@@ -37,10 +37,15 @@ public AstarteMutualTLSParametersFactory(IAstarteCryptoStore cryptoStore) : base
                 {
                     cryptoStore.GetCertificate()
                 },
-                IgnoreCertificateChainErrors = cryptoStore.IgnoreSSLErrors,
-                IgnoreCertificateRevocationErrors = cryptoStore.IgnoreSSLErrors,
                 SslProtocol = System.Security.Authentication.SslProtocols.Tls12,
-                AllowUntrustedCertificates = cryptoStore.IgnoreSSLErrors
+                CertificateValidationHandler = o =>
```

Comment:
Return `true` it means `Do you use the default certificate validator`?

Reply:
CertificateValidationHandler is from v4, so I deleted that validation now.